os/booting-on-launchtimevps.md

```diff
@@ -0,0 +1,133 @@
+# Running CoreOS on RimuHosting's LaunchtimeVPS service
+
+RimuHosting's LaunchtimeVPS service provides hosted virtual machines.  RimuHosting provide CoreOS as one of their standard images.  Customers can launch these VMs via the web interface or via the RimuHosting server management API.
+
+## About the RimuHosting CoreOS setup
+
+RimuHosting automatically creates the systemd networking files (no need to put networking setup in the cloud init block).
+
+The CoreOS image RimuHosting provide is from the stable channel (updating versions from time to time).
```

Comment:
The provided image is from the CoreOS `stable` channel and is updated with each stable release.